lang/strings_english.txt

```diff
@@ -362,6 +362,18 @@ $s_update_user_button = 'Update User';
 $s_verify_warning = 'Your account information has been verified. The account confirmation message you have received is now invalid.';
 $s_verify_change_password = 'You must set a password here to allow you to log in again.';
 
+# API tokens page
+$s_api_tokens_link = 'API Tokens';
+$s_api_token_create_form_title = 'Create API Token';
+$s_api_token_create_button = 'Create API Token';
+$s_api_token_name = 'Token Name';
+$s_api_token_disclose_message = 'Token to use for accessing API.  Note that this token will only be displayed once and spaces are ignored.';
```

Comment:
Mentioning spaces no longer needed.
Maybe show in red/bold that the token is displayed only one time.